src/lib/ThumbnailsSidebar.js

```diff
@@ -232,9 +232,9 @@ class ThumbnailsSidebar {
      * @return {HTMLElement} - thumbnail anchor element
      */
     createThumbnailNav() {
-        const thumbnailNav = document.createElement('a');
+        const thumbnailNav = document.createElement('button');
```

Comment:
Does `button` work better than `a` in this case?

Reply:
Anchors without an `href` attribute are considered placeholders rather than links and aren't included in the tab order ([source](https://www.w3.org/TR/2011/WD-html5-20110525/text-level-semantics.html#the-a-element)). Since we just want to response to a user action rather than navigating, a button seems more semantically correct and also obviates the need for `tabindex="0"`